### PR TITLE
Added multiple assignment and test case

### PR DESCRIPTION
**Pull request by rtsao**
This PR adds multiple variable assignment equivalent to the Sass syntax:

http://sass-lang.com/documentation/file.SASS_REFERENCE.html#each-multi-assign

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,40 +14,52 @@ function checkParams(params) {
...
   };
 }
 
 function processRules(rule, params) {
-  const values = {};
+  rule.nodes.forEach(node => {
 
-  rule.nodes.forEach((node) => {
-    params.values.forEach((value, index) => {
+    for (var i = 0; i < params.values[0].length; i++) {
```
Comment (outpunk) [inline index.js]: It would be nice to replace this with `forEach`.

Comment (rtsao) [inline index.js]: You're right, just updated the diff.

**Comment by outpunk** [comment]
Great job, thanks!

**Comment by outpunk** [comment]
Released.

**Pull request merged**